wiki/osu!api/pl.md

```diff
@@ -1,15 +1,18 @@
 ---
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
+no_native_review: true
 ---
 
 # osu!api
 
 osu! posiada publiczne [API](https://en.wikipedia.org/wiki/API), którego używać mogą zewnętrzne usługi. W przeszłości miały do niego dostęp tylko upoważnione osoby, ale obecnie każdy użytkownik osu! może z niego korzystać.
 
-Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!.
+Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!. Miej na uwadze, że dokumentacja zawarta na wiki osu!api nie jest w pełni kompletna czy wyczerpująca, a samo API nie jest obecnie rozwijane. 
 
-Miej na uwadze, że dokumentacja zawarta na wiki osu!api nie jest w pełni kompletna czy wyczerpująca, a samo API nie jest obecnie rozwijane. Nowa wersja API jest w trakcie powstawania na [repozytorium osu-web](https://github.com/ppy/osu-web).
+---
+
+Nowa wersja API jest w trakcie powstawania na [repozytorium osu-web](https://github.com/ppy/osu-web) i celuje w bycie bardziej stabilną <!--I'm not sure how to translate the word "robust"--> niż oryginał. Dokumentacja (nie zawsze aktualna) może być znaleziona na jej [stronie tytułowej](https://docs.ppy.sh).
+
+Autoryzacja dla nowego API następuje przy użyciu [protokołu OAuth 2](https://oauth.net/2/). Klienty <!--Not sure if it's "Klienci" or "Klienty" --> mogą być zarządzane na [stronie z ustawieniami konta](https://osu.ppy.sh/home/account/edit).
```

Comment:
[Both versions are apparently allowed](https://rjp.pan.pl/index.php?option=com_content&view=article&id=676:klienci-agenci-czy-klienty-agenty&catid=44&Itemid=145), but the passive voice here seems a bit roundabout, too. Maybe

```suggestion
Autoryzacja dla nowego API następuje przy użyciu [protokołu OAuth 2](https://oauth.net/2/). Klientami można zarządzać na [stronie z ustawieniami konta](https://osu.ppy.sh/home/account/edit).
```